src/noisepy/seis/plotting_modules.py

```diff
@@ -32,37 +32,44 @@
 #############################################################################
 # #############PLOTTING FUNCTIONS FOR FILES FROM S0##########################
 #############################################################################
-def plot_waveform(sfile, net, sta, freqmin, freqmax, savefig=False, sdir=None):
+def plot_waveform(
+    raw_data_store: RawDataStore, ts: DateTimeRange, net, sta, freqmin, freqmax, savefig=False, sdir=None
+):
     """
     display the downloaded waveform for station A
 
     PARAMETERS:
     -----------------------
-    sfile: containing all wavefrom data for a time-chunck in ASDF format
+    raw_data_store: Store to read Raw data from
+    ts: Timespan to plot
     net,sta,comp: network, station name and component
     freqmin: min frequency to be filtered
     freqmax: max frequency to be filtered
+    savefg: Whether to save the figures as a PDF on disk
+    sdir: Save directory
 
     USAGE:
     -----------------------
-    plot_waveform('temp.h5','CI','BLC',0.01,0.5)
+    plot_waveform(raw_data_store, '2023-10-21T00:00:00+0000 - 2023-10-21T12:00:00+0000', 'CI','BLC',0.01,0.5)
```

Comment:
Let's use a DateTimeRange object in the example, e.g.: `DateTimeRange("2015-03-22T10:00:00+0000", "2015-03-22T10:10:00+0000")`

Reply:
done